### DISCUSSION TITLE
P2G causing password reset requests on Garmin

### DISCUSSION THREAD

**Discussion by N6MIK** [philosowaffle/peloton-to-garmin#474] [Q&A]
I have P2G running in Docker on a Synology NAS. Everything is working great except... every 7 days, the Garmin Connect website is seeing password reset requests. They hit every 30 minutes until Garmin 'gives up' and stops sending me reset requested emails. To get back into Connect, I need to reset my password via another device. 
How to I prevent this from happening? Thank you!

**Comment by N6MIK** [comment]
Should update with this new info: I have sync set to 30 mins. The password reset emails come exactly 30 mins apart, and 7 days after I start the container. The docker log contains “missing Garmin auth cookie” in each sync attempt starting that coincides with a reset email. No other info in the log about failed auth (previous sync log is normal). Hope this helps.

**Comment by philosowaffle** [comment]
This projects [official recommendation](https://philosowaffle.github.io/peloton-to-garmin/configuration/json.html#app-config) is for folks to have P2G sync at most once a day.

Setting your sync frequency to 1hr or less can cause you account problems with both Garmin and Peloton.
